### PR TITLE
[AF-2792] Migrate treemap-recharts to new API

### PR DESCRIPTION
**Pull request by manaswinidas**
Migrate treemap-recharts to new API

### CODE REVIEW

File: packages/treemap-recharts/package.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
```
Comment (jesuino) [inline packages/treemap-recharts/package.json:47]: is this needed?

File: packages/treemap-recharts/src/Treemaps.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  fill?: string;
+  animationBegin?: number;
+  animationDuration?: number;
+  animationEasing?: AnimationEasingType;
+}
+
+export interface TreemapState {}
+
+export function Treemaps(props: TreemapProps) {
+  console.log(props.data);
```
Comment (jesuino) [inline packages/treemap-recharts/src/Treemaps.tsx:20]: Can we remove this log line?

File: packages/treemap-recharts/src/serviceWorker.ts [hunk 3]
```diff
@@ -0,0 +1,149 @@
+// This optional code is used to register a service worker.
```
Comment (jesuino) [inline packages/treemap-recharts/src/serviceWorker.ts:1]: Is this file needed?

File: packages/treemap-recharts/src/setupTests.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```
Comment (jesuino) [inline packages/treemap-recharts/src/setupTests.ts:1]: is this file being usedw

**Review by jesuino — CHANGES_REQUESTED** [review]
Hello @manaswinidas 

The component looks good to me, thanks!

I have a few questions for your in my review, would you please check it?

THanks!

**Comment by manaswinidas** [comment]
Sorry for not cleaning up earlier 😅

**Review by jesuino — APPROVED** [review]
LGTM, thanks!

**Pull request merged**